### ISSUE TITLE
好像不行了，浏览器打开能下载，但是直接调用链接下载的话不行

### ISSUE THREAD

**Issue by 838514984** [hanximeng/LanzouAPI#27]
<!DOCTYPE HTML>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=1.0,user-scalable=0" />
<script type="text/javascript" src="https://assets.woozooo.com/assets/includes/js/jquery.js"></script>
</head>
<body>
<style>
body{font-size: 16px;margin: 0;font-family: \5FAE\8F6F\96C5\9ED1;color: #333;background: #fff; border-color:transparent;-webkit-appearance: none;-webkit-tap-highlight-color:rgba(0,0,0,0);-webkit-tap-highlight-color:rgba(0,0,0,0.0);}
a{text-decoration: inherit;color: #999;}
.box{width:260px;margin:auto;    text-align: center;}
.box1{    margin-top: 35%;}
.box3{    margin-top: 50px;}
.box2_2{    padding: 25px;}
.submit{    width: 90%;
    margin: auto;
    height: 42px;
    line-height: 42px;
    border: 1px solid #d5d5d5;
    cursor: pointer;
        box-shadow: 0 1px 2px rgba(0, 0, 0, 0.075);border-radius: 2px;    color: #777;}
.submit:hover{    opacity: .7;}
.box2_1{    width: 60px;
    height: 60px;
    font-size: 30px;
    line-height: 60px;
    border-radius: 50%;
    background: #09f;
    color: #fff;
    margin: auto;}
#ok{background: #d4ffa3;
    width: 60px;
    height: 50px;
    border-radius: 50%;
    margin: auto;
    padding-top: 10px;    display: none;}
.ok{    height: 20px;display: none;
    width: 30px;
    border-left: 2px solid #84c33c;
    border-bottom: 2px solid #84c33c;
    transform: rotate(-45deg);
    margin-left: 15px;   }
#ok2{display: none;}
.goto{    border: 1px solid #fff;    color: #fff;box-shadow: inherit;
    background-color: #fff;
    background: #fff;}
#top{    font-size: 22px;padding: 20px;}
@media (max-width: 768px){
	.box{width:100%}
	#top{font-size: 0;padding: 20px;}
	}

.load2 .loader,
.load2 .loader:before,
.load2 .loader:after {border-radius: 50%;}
.load2 .loader:before,
.load2 .loader:after {position: absolute;content: '';}
.load2 .loader:before {width: 16px;height: 32px;background: #fff;border-radius: 32px 0 0 32px;top: -1px;left: -1px;transform-origin: 16px 16px;animation: load2 2s infinite ease 1.5s;}
.load2 .loader {font-size: 11px;text-indent: -99999em;margin: auto;position: relative;width: 30px;height: 30px;box-shadow: inset 0 0 0 4px #09f;border-radius: 50%;}
.load2 .loader:after {width: 16px;height: 32px;background: #fff;border-radius: 0 31px 31px 0;top: -1px;left: 15px;transform-origin: 0px 16px;animation: load2 2s infinite ease;}
@keyframes load2 {
  0% {
    transform: rotate(0deg);
  }
  100% {
    transform: rotate(360deg);
  }
}
.load2{    position: fixed;
    width: 100%;
    background: #fff;
    height: 100%;
    z-index: 999;
    top: 0;
    text-align: center;    padding-top: 100px;}
.loader2{    padding-top: 30px;}
</style>
<div id="top">下载文件</div>
<div class="load2" id="load2"><div class="loader"></div><div class="loader2">努力加载中...</div></div>

<div class="box">
<div id="box1" class="box1">
<div id="ok"><div class="ok"></div></div>
<div id="ok2" class="box2_2">已经通过验证</div>
<div id="go" class="submit goto"><div onclick="down_r(1);">验证并下载</div></div>
</div>

<div id="box2" class="box2">
<div class="box2_1">!</div>
<div class="box2_2">系统发现您的网络异常，需要验证后下载文件</div>
<div id="sub" class="submit"><div onclick="down_r(2);">验证并下载</div></div>
</div>

<div id="box3" class="box3">
<div id="sub2" class="submit goto"><div onclick="down_r(3);">验证并下载</div></div>
</div>

</div>
<script type="text/javascript">
	setTimeout('document.getElementById("load2").style.display="none";',2000);
	function down_r(el){
		$('#sub').text("提交中..."); 
		$.ajax({
			type : 'post',
			url : 'ajax.php',
			data : { 'file':'B29RcgsiV3MBJFQ4CnEBL1duAzdeMQQ0AD5SelBoVjpQIAIxDj4OKgB4U2YKMQJuA3YDNF5lA2BQMlE0UmsDOwc+UTYLZVcwAWZUNwo9AWRXOgM0XjgEZwAxUmZQNVZvUGICNw5kDjkAYFNnCmkCNANuAyleNwM2UDNRNlI9AzoHM1EzC3lXMQFnVDAKawEvVz0DNV4uBDcAMFJ7UG1WYlAzAjcOYQ5tADRTZwptAmIDPQMyXmADNVA1UWVSOwMwBz5RNgswV2ABNFRmCmYBZVdrA2deMARnADBSNlAiVi1QOQIlDmwOPAA4UzgKMQItAyMDb15x','el':el,'sign':'V2ZQMAtiVTMGYwZlCmxSZFQ+DztWaA==' },
			dataType : 'json',
			success:function(msg){
				if(msg.zt == '1'){
					$("#box2").hide(150);
					$("#ok").show(150);
					$("#ok2").show(150);
					$(".ok").show(500);
					$("#go").removeClass("goto");
					$("#go").html("<a href="+ msg.url +"><span class=tc2>立即下载</span></a>");
				}else{
					//$('#sub').val("验证并下载"); 
					setTimeout('window.location.reload();',1000);
				}
			},
			error:function(){
				//$("pwderr").text("获取失败，请重试");
			}
	
		});
	}
</script>
</body>
<div style="display:none"></div>
</html>

**Comment by hanximeng** [comment]
已修复，反馈只需要附上链接即可，我会尝试复现问题
